books/free-programming-books-langs.md

```diff
@@ -471,6 +471,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [The little book about OS development](http://littleosbook.github.io) - Erik Helin, Adam Renberg
 * [The New C Standard - An Economic and Cultural commentary (2009)](http://www.knosof.co.uk/cbook/cbook.html) - Derek M. Jones (PDF)
 * [TONC GBA Programming - Game Boy Advance Development](http://www.coranac.com/tonc/text/toc.htm)
+* [C Programming Notes](https://goalkicker.com/CBook/) - CNotesForProfessionals
```

Comment:
```suggestion
```

Reply:
If the only change is a duplicate, would it make more sense to just close this PR? The suggested change would bring the diff to nothing.

Reply:
> If the only change is a duplicate, would it make more sense to just close this PR? The suggested change would bring the diff to nothing. 

It would but I don't have that power.